frontend/pages/claim.tsx

```diff
@@ -33,6 +34,26 @@ export default function Claim() {
     setButtonLoading(false); // Toggle
   };
 
+
+  const addToWallet = async() => {
+    try  {
+      if (provider != null) {
+        await provider.send(
+          'wallet_watchAsset',{type: 'ERC20',
```

Comment:
Damn, thought that would be an easy one.
in provider.send, the compiler expects the second paramter to be an array, but metamask only works with an object.
I didn't notice it, because it worked in debug mode and forgot to recompile.

I tried to fix it, but there is no new ethers version, that supports that use case and I was not able to circumvent the underlying providers. 
So it seems we need to revert it.